packages/pigeon/lib/ast.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:collection/collection.dart' show ListEquality;
 import 'package:meta/meta.dart';
+import 'kotlin_generator.dart';
```

Comment:
I can't add this to the normal kotlin options class because these options are used for each `@ProxyApi` class. The normal kotlin options is used for the whole file. I changed it to 

```dart
import 'kotlin_generator.dart' show KotlinProxyApiOptions;
```

though.